modules/adding-to-a-project.adoc

```diff
@@ -6,18 +6,21 @@
 [id="adding-to-a-project_{context}"]
 = Adding to a project
 
-.Procedure
+You can add items to your project by using the *+Add* page in the *Developer* perspective.
+
+.Prerequisites
 
-. Select *Developer* from the context selector at the top of the web console
-navigation menu.
+* You have created a project.
+
+.Procedure
 
-. Click *+Add*
+. In the *Developer* perspective, navigate to the *+Add* page.
 
-. At the top of the page, select the name of the project that you want to add to.
+. Select your project from the *Project* drop-down menu.
```

Comment:
```suggestion
. Select your project from the *Project* menu.
```
Project menu or project list

Reply:
_*Project* drop-down menu_ was used throughout this assembly. But, I think you are right. I updated a few other modules.